scripts/get-shapefiles.py

```diff
@@ -0,0 +1,277 @@
+        step += 1
+        zip = zipfile.ZipFile('data/land-polygons-split-3857.zip')
+        zip.extractall("data")
+        print(". Done.\n")
+    
+    #5 antarctica-icesheet-polygons-3857
+    if args.option_aip and (not args.option_no_download or not os.path.isfile("data/antarctica-icesheet-polygons-3857.zip")):
+        print(str(step) + ". Downloading antarctica-icesheet-polygons-3857...")
+        step += 1
+        url = 'http://data.openstreetmapdata.com/antarctica-icesheet-polygons-3857.zip'
+        file_name = download_file(url, "data")
+    
+    if args.option_aip and not args.option_no_extract:
+        print(str(step) + ". Expanding antarctica-icesheet-polygons-3857...", end="")
+        step += 1
+        zip = zipfile.ZipFile('data/antarctica-icesheet-polygons-3857.zip')
+        zip.extractall("data")
+        print(". Done.\n")
+    
+    #6 antarctica-icesheet-outlines-3857
+    if args.option_aio and (not args.option_no_download or not os.path.isfile("data/antarctica-icesheet-outlines-3857.zip")):
+        print(str(step) + ". Downloading antarctica-icesheet-outlines-3857...")
+        step += 1
+        url = 'http://data.openstreetmapdata.com/antarctica-icesheet-outlines-3857.zip'
+        file_name = download_file(url, "data")
+    
+    if args.option_aio and not args.option_no_extract:
+        print(str(step) + ". Expanding antarctica-icesheet-outlines-3857...", end="")
+        step += 1
+        zip = zipfile.ZipFile('data/antarctica-icesheet-outlines-3857.zip')
+        zip.extractall("data")
+        print(". Done.\n")
+    
+    print()
+    
+    #index
+    
+    if not args.option_quiet_mode and not args.option_no_shape:
+        print("Going to index shapefiles...")
+        try:
+            input("Press Enter to continue...")
+        except Exception:
+            pass 
+    else:
+        print("Indexing files...\n")
+    
+    if not args.option_no_shape:
+        if args.option_wb:
+            subprocess.call("shapeindex --shape_files data/world_boundaries/world_bnd_m.shp")
+            print()
+            subprocess.call("shapeindex --shape_files data/world_boundaries/builtup_area.shp")
+            print()
+            subprocess.call("shapeindex --shape_files data/world_boundaries/places.shp")
+            print()
+            subprocess.call("shapeindex --shape_files data/world_boundaries/world_boundaries_m.shp")
+            print()
+        if args.option_swp:
+            subprocess.call("shapeindex --shape_files data/simplified-land-polygons-complete-3857/simplified_land_polygons.shp")
+            print()
+        if args.option_ne:
+            subprocess.call("shapeindex --shape_files data/land-polygons-split-3857/land_polygons.shp")
+            print()
+        if args.option_wps:
+            subprocess.call("shapeindex --shape_files data/antarctica-icesheet-polygons-3857/icesheet_polygons.shp")
+            print()
+        if args.option_aip:
```

Comment:
@tilmanb: Thanks, I will correct this and will also change the water acronyms to land